mkversion.sh

```diff
@@ -48,11 +48,11 @@ if command -v git >/dev/null && [ -d "$(dirname "$0")/.git" ] ; then
     # dirty and append that last, including it here will make dirty trees 
     # directly on top of tags show up with version_from_git as 2.46-dirty which
     # will not match 2.46 from the changelog and then result in a final version
-    # like 2.46+git2.46.2.46 which is silly and unhelpful
+    # like 2.46+2.46.2.46 which is silly and unhelpful
     # tracking the dirty independently like this will produce instead 2.46-dirty
-    # for a dirty tree on top of a tag, and 2.46+git83.g1671726-dirty for a 
+    # for a dirty tree on top of a tag, and 2.46+83.g1671726-dirty for a
```

Comment:
would it make sense to move the g in front?

2.46+g83.1671726-dirty

For people wondering I at some point chased this and the g there both stands for g(it) but it also a version marker for the format of the hash in a way

Reply:
I can look into that. I honestly didn't notice the second g there at all.